libs/shared/src/entities/layout/layout.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { EnvironmentId, LayoutId, LayoutName, OrganizationId, UserId } from '../../types';
+import { ChannelTypeEnum, ITemplateVariable } from '../message-template';
+
+export interface ILayoutEntity {
+  _id?: LayoutId;
+  _organizationId: OrganizationId;
+  _environmentId: EnvironmentId;
+  _creatorId: UserId;
+  name: LayoutName;
+  channel: ChannelTypeEnum;
+  content: string;
+  contentType: string;
+  variables?: ITemplateVariable[];
+  isDefault: boolean;
+  isDeleted: boolean;
+  createdAt?: string;
+  updatedAt?: string;
+}
```

Comment:
Let's make this shared and call it DTO (data transfer object) as entity we are using it more for the Repository objects. What do you think? Also like this we will have it common for both.
Unless if for your needs you prefer to have a separate object but might be more difficult to keep in sync with backend changes. Up to your preference. 🙂

Reply:
I actually did that as a dto initially. But then felt its odd to use dto in the web. I am actually not sure what is our practice here. We seem to not have consistency about that. 
I'll do a bit more research here

Reply:
@davidsoderberg wdyt?

Reply:
I am merging this for now